jib-core/src/test/java/com/google/cloud/tools/jib/builder/steps/BuildAndCacheApplicationLayerStepTest.java

```diff
@@ -59,7 +59,7 @@
   private Path temporaryCacheDirectory;
 
   @Before
-  public void setUp() throws IOException, URISyntaxException {
```

Comment:
Sorry this is extraneous to this PR: this exception isn't actually thrown and flags a warning.